docs/package.json

```diff
@@ -4,13 +4,14 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "build-docs": "pnpm run update-contributors && vitepress build && cp _redirects .vitepress/dist",
+    "build-docs": "pnpm run update-contributors && pnpm run -w build && vitepress build && cp _redirects .vitepress/dist",
```

Comment:
@kerwanp Probably worth explaining this weird little quirk of docs:

- Turborepo will just run the `build` script in every package. We recommend contributors do this when setting up the project locally
- However, because we have our contributors list pulling from GitHub, we need an API token to pull this data
  - To fix this, either we require everyone to have a GitHub API token (annoying), or we just make the docs build a different command (less annoying)
- Because the docs command is intentionally different, this means it runs at different times than the build
- The docs before didn’t need a build of all packages, so we just skipped it. But since that’s now a requirement, we’ll just build as part of the docs.

This may add a tiny bit of delay to the docs, but the monorepo builds really quickly, so it’s better to just build everything than try and surgically determine the minimal build. It’s always a balance of “what’s efficient compute-wise” and “what’s easiest to work with people-wise” and it’s always good to try and satisfy both reasonably.